dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -136,70 +141,107 @@ public String getTitle(){
 		}
 	}
 
-
-
+    @Override
     public String getVersionId() {
     	return getIdentifier();
     }
 
+    @Override
     public String getVersionType() {
     	return new String("content");
     }
 
+    @Override
     public void setVersionId(String versionId) {
     	setIdentifier(versionId);
     }
 
+    @Override
     public String getInode() {
-    	if(InodeUtils.isSet((String) map.get(INODE_KEY)))
+    	if(InodeUtils.isSet((String) map.get(INODE_KEY))) {
     		return (String) map.get(INODE_KEY);
-
+    	}
     	return "";
     }
 
+    /**
+     * 
+     * @param inode
+     */
     public void setInode(String inode) {
         map.put(INODE_KEY, inode);
     }
 
+    /**
+     * 
+     * @return
+     */
     public long getLanguageId() {
     	return (Long)map.get(LANGUAGEID_KEY);
     }
 
+    /**
+     * 
+     * @param languageId
+     */
     public void setLanguageId(long languageId) {
         map.put(LANGUAGEID_KEY, languageId);
     }
 
+    /**
+     * 
+     * @return
+     */
     public String getContentTypeId() {
       return (String) map.get(STRUCTURE_INODE_KEY);
     }
-    /**@deprecated as of 4.1 
+
+    /**
+     * @deprecated as of 4.1 
      * use instead:
      * {@link #getContentTypeId()}
      */
-    @Deprecated 
     public String getStructureInode() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @Deprecated annotation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)